routers/api/v1/api.go

```diff
@@ -610,6 +612,7 @@ func RegisterRoutes(m *macaron.Macaron) {
 			m.Group("/members", func() {
 				m.Get("", org.GetTeamMembers)
 				m.Combo("/:username").
+					Get(org.GetTeamMember).
```

Comment:
permission check.

Reply:
What permission check are you suggesting to add? I think we are already checking for org membership?